spec/Event/ImportEventSpec.php

```diff
@@ -13,7 +13,7 @@ class ImportEventSpec extends ObjectBehavior
 {
     function let()
     {
-        $this->beConstructedWith('foobar');
+        $this->beConstructedWith('foo', 'bar');
```

Comment:
Oh, I just merged my pr of log event fixes. And I did similar updates to `ImportEvent` and `ImportCommand` there. But I did it slightly different, and added a little more to `ImportEvent`. Sorry about this, but you have to manage this conflict.. One should do "one change, one pr", as this proves!